requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django>=1.6
+Django==1.8.7
```

Comment:
I'd like the travis build matrix to be soemthing like {2.7, 3.4, 3.5} * django{1.7, 1.8, 1.9}, but that should probably come as a separate PR.